reporting/api/src/main/java/org/jboss/windup/reporting/severity/DuplicateIssueCategoryException.java

```diff
@@ -0,0 +1,16 @@
+package org.jboss.windup.reporting.severity;
```

Comment:
is *severity* now category, so why we have package as `org.jboss.windup.reporting.severity` and not `org.jboss.windup.reporting.category`?